Client/Frontend/Browser/Tabs/TabCollectionViewController.swift

```diff
@@ -0,0 +1,93 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import UIKit
+
+class TabCollectionViewController: UIViewController,
+                                   Themeable {
+    struct UX {}
+
+    var notificationCenter: NotificationProtocol
+    var themeManager: ThemeManager
```

Comment:
> `var themeManager: ThemeManager`

Will this eventually be replaced by state updates from Redux store? At that point I'm guessing we won't need to inject the `themeManager` directly?

Reply:
Great question, I'm not really sure what would be best for this. For the vast majority of cases this is just a static store of colors that don't change because users tend not to change their theme settings very often. I think since it is technically variable it probably should live in the redux store from a philosophical perspective but adding this to the store will be a lot of overhead for very little benefit. I could go either way, do you or @yoanarios have any opinions or preferences?

Reply:
To be honest I didn't think of theming as part of Redux. I understand the point where changes in theme are actions and could become part of the Store state but I'm not sure if the gain overpasses the pains. ThemeManager integration is a project we recently finished so it feels weird to change it once again but if we feel strongly about adding it I can't think of a good reason to go against it.

Reply:
I don't think the theming refactor work would be a total waste, it would just add an extra step where the applyTheme method takes a model that comes from the store instead of the theme itself but it really does feel like overkill without much benefit. Redux really shines where it's resolving multiple types of state change that have interactions with one another but theme changes don't impact anything other than what color lives under a color token. Even state that might impact color, like an error state will refer to it by token so it wouldn't even have an impact there. 🤷‍♀️